analyzer.go

```diff
@@ -65,6 +66,9 @@ func evalConstString(expr ast.Expr, info *types.Info) string {
 		}
 		return "%s"
 	case *ast.Ident:
+		if typ, ok := info.Types[v]; ok {
+			return constant.StringVal(typ.Value)
```

Comment:
I think this will panic for non-constant identifiers; can you test it against e.g. `foo := "bar"` rather than `const foo = "bar"`?

Checking for `typ.Value != nil` should suffice to fix it